Mage/Module.cs

```diff
@@ -126,17 +160,12 @@ public void SetupUsercommands(Action<Exception> cb)
 			commandHandlerActions = new Dictionary<string, Action<JObject, Action<Exception, JToken>>>();
 			commandHandlerFuncs = new Dictionary<string, Func<JObject, UserCommandStatus>>();
 
-			if (Commands == null)
```

Comment:
I believe this is correct. This is also why we switch from `virtual` to ` abstract` and cease to provide a default definition. `virtual` appear to be problematic for strong type.